### PR TITLE
Mention warning and error message docs in PR template

### LINKED ISSUE #7580: PR template should remind PR authors to document warnings and errors
See https://github.com/coq/coq/pull/7559#issuecomment-390749207 and https://github.com/coq/coq/pull/7559#issuecomment-390872924.  See also https://github.com/coq/coq/issues/7373, which is broader in scope.

### PR DESCRIPTION
**Pull request by JasonGross**
This closes #7580

c.f. https://github.com/coq/coq/pull/7559#issuecomment-390749207 and
https://github.com/coq/coq/pull/7559#issuecomment-390872924.  This
should be reverted if and when we move to autogenerated docs for
warnings and errors, as suggested in #7373.

**Kind:** documentation / infrastructure.

Fixes / closes #7580

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -12,5 +12,5 @@ Fixes / closes #????
 
 <!-- If this is a feature pull request / breaks compatibility: -->
 <!-- (Otherwise, remove these lines.) -->
-- [ ] Corresponding documentation was added / updated.
+- [ ] Corresponding documentation was added / updated (including any warning- and error-messages added / removed / modified).
```
Comment (Zimmi48) [inline .github/PULL_REQUEST_TEMPLATE.md]: You're the native English speaker, not me, but I'm surprised by the use of dashes here.

**Review by Zimmi48 — APPROVED** [review]
Welcome addition. Thanks!

File: .github/PULL_REQUEST_TEMPLATE.md (hunk 1 above)
Comment (JasonGross) [reply .github/PULL_REQUEST_TEMPLATE.md]: I think it works fine either way; I'll update it.

**Comment by Zimmi48** [comment]
I'm going to merge this. Would you mind squashing the two commits together though?

**Pull request merged**